CONTRIBUTING.md

```diff
@@ -77,10 +77,11 @@ To send us a pull request, please:
 2. Modify the source; please focus on the specific change you are contributing.
     - Please follow the instructions in **Building** to ensure your changes will be formatted properly (`npm install` installs a code formatter)
 3. Ensure local tests pass.
-4. Make sure to add tests to your change
-5. Commit to your fork using clear commit messages.
-6. Send us a pull request, answering any default questions in the pull request interface.
-7. Pay attention to any automated CI failures reported in the pull request, and stay involved in the conversation.
+4. Once you are done with your change, run `npm run newChange`, follow the prompts, then commit the changelog item to your fork.
```

Comment:
What do you think about adding this command into the PR template (handy reminder "what was that command I should run?")

Reply:
in the pr template we have "I have read contributing", I don't think we need it in both places